pkg/util/containers/pause.go

```diff
@@ -22,5 +24,17 @@ func IsPauseContainer(labels map[string]string) bool {
 	// they only have io.kubernetes.pod.name
 	// See https://github.com/containerd/cri/issues/922
 	_, podFound := labels[podNameLabel]
-	return !ctrFound && podFound
+	isPauseContainerByLabels := !ctrFound && podFound
```

Comment:
maybe we can add a short path here and  return if `isPauseContainerByLabels` is `true`